reaper.go

```diff
@@ -52,7 +52,8 @@ func NewReaper(ctx context.Context, sessionID string, provider ReaperProvider, r
 		BindMounts: map[string]string{
 			"/var/run/docker.sock": "/var/run/docker.sock",
 		},
-		AutoRemove: true,
+		AutoRemove:      true,
+		SkipLogProducer: true,
```

Comment:
What do you think about making `SkipLogProducer` true by default? Then existing client code won't have to change anything on their side.

Reply:
I like that idea.  If the user doesn't need it, it would save resources too (it wouldn't spin up goroutines that have to be managed if they aren't requested).  I will update 👍